pages/_app.jsx

```diff
@@ -24,15 +25,17 @@ const DynamicWalletContextProvider = dynamic(
   { ssr: false }
 )
 
-function MyApp({ Component, pageProps }) {
+const JotaiProvider = dynamic(() => import('jotai').then((jt) => jt.Provider), { ssr: false })
```

Comment:
Interesting. You found that you needed the nextjs dynamic imports for the jotai provider? In the other app I was able to use:

`import { Provider } from 'jotai'`

and `<Provider>` as the first element in my _app.jsx. I wonder why we got different results?

Reply:
Hmm yeah I was getting an error, maybe I needed to restart the server. I just switched it to `import { Provider } from 'jotai'` and it works fine.